src/_includes/google_tag_manager.html

```diff
@@ -0,0 +1,6 @@
+<!-- Google Tag Manager (noscript) -->
```

Comment:
I think the file name is misleading because it suggests that this is all of Google Tag Manager. 
It should mention "noscript". 

`google_tag_manager_noscript.html` or something similar

Reply:
I changed the name to `google_tag_manager_noscript.html`